pkgs/development/libraries/intel-media-driver/default.nix

```
@@ -32,4 +35,9 @@ stdenv.mkDerivation rec {
     platforms = platforms.linux;
     maintainers = with maintainers; [ jfrankenau ];
   };
+
+  postFixup = stdenv.lib.optional enableX11 ''
```

```suggestion
  postFixup = stdenv.lib.optionalString enableX11 ''
```

Because `optional` is for lists.